screens/NotificationsScreen.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+import * as firebase from 'firebase';
+
+export default class NotificationsScreen extends React.Component {
+  static navigationOptions = {
+    header: null
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      notifications: [],
+    };
+  }
+
+  async componentDidMount() {
+    var notifications = [];
+    await firebase
+      .database()
+      .ref('notification')
+      .orderByChild("timestamp")
+      .limitToLast(10)
+      .once('value')
+      .then(function(data) {
+        data.forEach(function(childNode) {
+          var notification = childNode.val();
+          notifications.unshift(notification);
+        });
+        this.setState({
+          notifications: notifications
+        });
+      }.bind(this));
+  }
+
+  parseTimestamp(timestamp) {
+    let month = timestamp.slice(5,7);
+    let date = timestamp.slice(8,10);
+    let hour = timestamp.slice(11,13);
+    let minute = timestamp.slice(14,16);
+    let ampm;
+    if (hour < 12) {
+      ampm = 'AM';
+    } else {
+      ampm = 'PM';
+    }
+    var months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];    
```

Comment:
Can we add a file in constants folder holding the months array?

In general, we should never be declaring objects that are the same over and over again.

Reply:
Oops starting review.